form4.php

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Form submit</title>
+</head>
+<body>
+  <!-- 1st assignment -->
+  <?php
+  // cond to check if post request
+    if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+      $f_name =  $_POST["fname"];
+      $l_name =  $_POST["lname"];
+      $file = $_FILES['file']['name'];
+      $mark = $_POST['marks'];
+      $pattern = "/[A-Za-z]+\|[0-9]+/";
+      $mob_no = $_POST['mob_no'];
+      $check = substr($mob_no,0,3);
+      // cond to check valid string or not
+      if ( empty($f_name) || empty($l_name)) {
+        echo "Name field is empty<br>";
+        include "php_assin4.php";
+      }
+      elseif ((ctype_alpha($f_name) == false) || (ctype_alpha($l_name) == false)) {
+        echo "Not a valid name";
+        include "php_assin4.php";
+      }
+      elseif (empty($file)) {
+        echo "Please include image file.";
+        include "php_assin4.php";
+      }
+      elseif ((empty($mark)) || (preg_match($pattern,$mark) == 0)){
+        echo "Please enter marks properly.<br>".$mark;
+        include "php_assin4.php";
+      }
+      // condition to check if indian and valid number or not
+      elseif ((strlen($mob_no) != 13) || ($check != "+91")) {
+        echo "Moble number is invalid.<br>";
+        include "php_assin4.php";
```

Comment:
Is it still valid if I add character in the number? Don't think so. Update that as well